.travis.yml

```diff
@@ -34,10 +36,13 @@ before_install:
   - sudo apt-get install -y libopencv-dev valgrind
 
 script:
+  # Checks for core C++
   - cd ../../core/
   - mkdir build && cd build
   - cmake -DCMAKE_CXX_COMPILER=$COMPILER .. && make
   - cd ..
   - valgrind --leak-check=full --error-exitcode=1 ./build/grf exclude:[characterization]
+  # Checks for R package
```

Comment:
Small potential improvement here: would it be possible to `lint` before `build`ing?